src/FirePokemon.java

```diff
@@ -0,0 +1,48 @@
+public class FirePokemon extends Pokemon {
+
+//    Could place an arraylist for all fire type pokemons??
+
+    private boolean effective;
+    private String sound;
+
+
+    public FirePokemon(String name, String type, int level, int xp, int hp, int attack, int special, int defence, int accuracy, double height,String sound) {
+        super(name, type, level, xp, hp, attack, special, defence, accuracy, height);
+        this.sound = sound;
```

Comment:
Je vult in je constructor geen `effective` in. Wat is ook al weer de standaard waarde van een boolean? Had je daar rekening mee gehouden?